build.sbt

```
@@ -116,3 +116,15 @@ val compilerOptions = Seq(
   "-Wunused:privates",
   "-Wvalue-discard"
 )
+
+// JaCoCo code coverage
+Test / jacocoReportSettings := JacocoReportSettings(
+  title = s"spark-metadata-tool Jacoco Report - scala:${scalaVersion.value}",
```

```suggestion
  title = s"spark-metadata-tool_${scalaVersion.value} Jacoco Report",
```
To follow scala standard